app/poc-common/clinical-services/mappers/createEncounterMapper.js

```diff
@@ -64,7 +63,6 @@
         if (field.field.concept.set) {
           var obs = {
             concept: field.field.concept.uuid,
-            obsDatetime: Bahmni.Common.Util.DateUtil.now(),
```

Comment:
What if the date is passed by the user in retrospective mode, test request and drug prescription, lab and pharmacy modules respectively?

Reply:
Those parts of the application do not use the createEncounterMapper right? We are planning to address that on a future PR, when the proper back-end support is added.

Reply:
@valvijo  and @edrisse , I have done these changes for pharmacy module , I'm just creating some unit tests, by the end of the day will commit these changes

Reply:
Got it